sympy/matrices/normalforms.py

```diff
@@ -0,0 +1,115 @@
+from sympy.matrices import *
```

Comment:
It is expected that this line be the first in all SymPy modules.

    from __future__ import print_function, division

Also, library code should not use `import *` but give an explicit list of imports.